tests/test_assessment_status.py

```diff
@@ -655,6 +656,24 @@ def test_boundary_in_progress_expired(self):
         a_s = AssessmentStatus(user=self.test_user, as_of_date=nowish)
         self.assertEquals(a_s.overall_status, 'Partially Completed')
 
+    def test_all_expired_old_tx(self):
+        self.login()
+        # backdate outside of baseline window (which uses consent date)
+        backdate, nowish = associative_backdate(
+            now=now, backdate=relativedelta(months=4, hours=1))
```

Comment:
Would you be able to use the current datetime instead of `now`?

Reply:
`now` is the effective current datetime, cached at the module level to avoid the indeterminate problems of test run time.

Reply:
That would make individual tests run consistently relative to eachother, but I don't think it would cover test runs on different days, like the situation that allowed us to discover the `dateutil.relativedelta` forward/back issue